Applications/QuickShow/QSWidget.cpp

```diff
@@ -40,6 +43,126 @@ QSWidget::~QSWidget()
 {
 }
 
+void QSWidget::clear()
+{
+    m_bitmap = nullptr;
+    m_path = {};
+
+    on_scale_change(100);
+    update();
+}
+
+void QSWidget::flip(FlipDirection flip_direction)
+{
+    auto w = m_bitmap->width();
+    auto h = m_bitmap->height();
+
+    auto new_bitmap = Gfx::Bitmap::create_purgeable(m_bitmap->format(), { w, h });
```

Comment:
Since we're not using the purgeable feature, let's just use Gfx::Bitmap::create()